js/exercise-00.js

```diff
@@ -0,0 +1,23 @@
+/* 
+   Create a constant variable of each of the following primitive types:
+   - String
+   - Number
+   - Boolean
+   - Null
+   - Undefined
+
+   Log out each variable to the terminal at the end.
+
+*/
+
```

Comment:
Have you ensured this script is properly working before pushing it? 🤔

⚠️ FIX: spot the bug and fix it